packages/knip/test/cli-reporter-markdown.test.ts

```diff
@@ -0,0 +1,31 @@
+import assert from 'node:assert/strict';
+import test from 'node:test';
+import { resolve } from '../src/util/path.js';
+import { execFactory } from './helpers/exec.js';
+
+const cwd = resolve('fixtures/module-resolution-non-std');
+
+const exec = execFactory(cwd);
+
+test('knip --reporter markdown', () => {
+  const markdown = `## Unused files (1)
+
+* src/unused.ts
+
+## Unlisted dependencies (2)
+
+| Symbol          | Location     |
+|-----------------|--------------|
```

Comment:
I didn't look at the rendered output, but if the cell contents are centered, perhaps we can right-align everything?

Reply:
The cell contents are also left-aligned. I think it's the best way anyways

Reply:
Oh my, of course I meant left-aligned, lol 🤦

Reply:
Example of rendered output:

# Knip report

## Unused files (1)

* src/unused.ts

## Unlisted dependencies (2)

| Name            | Location     |
|:----------------|:-------------|
| unresolved      | src/index.ts |
| @org/unresolved | src/index.ts |

## Unresolved imports (1)

| Name         | Location     |
|:-------------|:-------------|
| ./unresolved | src/index.ts |